### ISSUE TITLE
"missing setup file"

### ISSUE THREAD

**Issue by ShashankBice** [whimian/pyGeoStatistics#1]
Hi, in the official documentation, it says that we can install using the setup.py after creating a dedicated environment. However, the setup.py file is not residing currently in the git repo.

**Comment by whimian** [comment]
@ShashankBice Sorry for the inconvenience. 

Actually you don't have to install it, just clone or download this repo and add it to python path.

```python
import sys
ppath = "PAHT TO pyGeoPressure"

if ppath not in sys.path:
    sys.path.append(ppath)
```

**Comment by ShashankBice** [comment]
Thanks! Just got confused by the documentation.

**Comment by whimian** [comment]
Fixed by: d695f40dbfed8a5cec76d48497c164b55dc7fa9a
